src/main/scala/fable/Consumer.scala

```diff
@@ -175,6 +175,25 @@ class Consumer[F[_]: ContextShift: Monad: Sync, K, V] private[fable] (
       }.toMap
     }
 
+  /**
+    * Get metadata about partitions for all topics that the user is authorized to view. This method will issue a remote call to the server.
+Specified by:
+    *
+    * @see [https://kafka.apache.org/21/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#listTopics--]
+    */
+  def listTopics(): F[Map[Topic, List[Partition]]] =
+    for {
+      topics <- eval(_.listTopics())
+    } yield {
+      topics.asScala.map {
+        case ((topic, partitionInfos)) =>
+          val partitions: List[Partition] =
+            partitionInfos.asScala.toList.map(partitionInfo =>
```

Comment:
When we directly call asScala on the java list that gets returned, we get a Buffer. Given the name of the method it felt better to return a list type. Any thoughts on if a Buffer would be useful here?

Reply:
I think you could also use `Seq` if you don't care about the particular type of sequence, and then the calling user can call `toList` if they need an operation that's only on list.

Reply:
Ah, so a sequence makes sense to us because it would easy to transform it a buffer, list or anything else that extends Sequence. Should we rename it since we're not returning a list? Are we trying to match the kafkaClients API exactly?

Reply:
I've been trying to stay really close to the kafka-clients API. My thinking is that it makes it easier to use if you already know kafka-clients, and you can easily look up the underlying behavior if you need to.